.tool-versions

```diff
@@ -1,5 +1,5 @@
 ruby 2.7.2
 python 3.9.7
-pre-commit 3.2.2
+pre-commit 3.6.2
```

Comment:
> ih-setup's own version needs to be updated.

@vanzin - own version of pre-commit? Or is "own" referring to something else?

Reply:
https://github.com/ConsultingMD/homebrew-ih-public/blob/master/meta/bump